.gitignore

```diff
@@ -1,3 +1,5 @@
+.vscode
```

Comment:
Was this intentional? I think it's ok but am wondering if it accidentally got picked up.

Reply:
I was seeing it in my `git status` and thought I'd add it to `.gitignore` so that I don't accidentally include it with `git add -A` which I use from time to time. I don't think we'll want to include vscode stuff in kurbo in the future but if we do we can always remove it from .gitignore :)